README.md

```diff
@@ -47,7 +47,8 @@ run();
 3. Bind component to DOM
 ```html
 <div data-nc="componentName"
-     data-nc-params-componentName="{}"></div>
+     data-nc-params-componentName="{}"
+     data-nc-loading="lazy"></div>
```

Comment:
No need for extended example here, since it's covered below. Let's keep initial example simple.